content/60.methods.md

```diff
@@ -182,106 +193,131 @@ Manubot users can write in the simple markdown format, render the manuscript in
+The trajectory of the pandemic itself also likely influenced participation: engagement waned over the course of the pandemic as labs opened back up and researchers were able to return to their work, and we recruited very few medical doctors to the project, which is unsurprising given the load on medical professionals during this time.
+Engagement that waxes and wanes is, in however, typical when writing massively open online papers [@doi:10.5334/kula.63].
+Adding features such as spell-check has improved user friendliness, and additional features such as automatically checking the formatting of citations could further improve the usability of this tool.
+In the future, a formal study of participation could allow for quantification of these biases and improved efforts to foster inclusion.
+
+Some other limitations are challenges associated with of massively open online papers in general.
+With such a large amount of text, it is not possible to keep all sections of the manuscript up to date.
+Readers are not able to distinguish when each section was updated, and even a functionality such as GitHub's blame feature would not distinguish minor changes from substantive updates to the text.
+Additionally, while much of the data and statistics update automatically, the text itself required updating by human experts.
+This asynchrony could potentially introduce incompatibility between the figures and the surrounding text.
+Similarly, in line with the collaboration-related challenges of the project, some authors returned to update their text, while others did not.
+As a result, the lead authors of each paper often spent several weeks prior to submission updating the text to reflect new developments in each area.
+In the future, it may be possible to streamline this process through integration with a tool such as CoronaCentral [@doi:10.1073/pnas.2100766118] to automatically identify relevant, high-impact papers that need to be included, although expertise would still be required to incorporate them.
+Another challenge involves tracking preprints as they are reviewed or critiqued, revised, and potentially published.
+While updating the content of the manuscript would likely fall to human contributors, automatic detection of published versions of preprints could be integrated in the future (as developed by [@doi:10.1101/2021.03.04.433874]).
+These challenges are exacerbated by the scale of the infodemic, but developing solutions would benefit future projects with more typical trends in literature.
+Similarly, outputting machine readable summaries of key information in the manuscripts developed by the Consortium could reduce our contribution to the infodemic.
+As it stands, the integration of Compact Uniform Resource Identifier does make a step in this direction, as it could be used to extract relationships among clinical trials, genes, publications, and other entities.
```

Comment:
```suggestion
As it stands, the integration of Compact Uniform Resource Identifier does make a step in this direction, as it could be used to extract relationships among clinical trials, genes, publications, and other entities.
Thus, the experience of using Manubot for a massive non-technical project has laid the foundation for future additions to enhance user experience and inclusivity.
```